pyisemail/diagnosis/dns_diagnosis.py

```diff
@@ -21,4 +21,5 @@ class DNSDiagnosis(BaseDiagnosis):
             "Couldn't find an MX record for this domain " "but an A record does exist."
         ),
         "NO_RECORD": "Couldn't find an MX record or A record for this domain.",
+        "NULL_RECORD": "Domain does not support email service."
```

Comment:
There's a small issue here that I will fix when rebasing. This needs to match the key above.